VoxelSpace.html

```diff
@@ -371,7 +374,37 @@
 }
 
 // ---------------------------------------------
-// The main render routine
+// Interpolate between two RGB colors.
+
+function ColorInterp(a, b, t)
+{
+    var a0 = (a >> 16) & 0xFF;
+    var a1 = (a >> 8) & 0xFF;
+    var a2 = a & 0xFF;
+
+    var b0 = (b >> 16) & 0xFF;
+    var b1 = (b >> 8) & 0xFF;
+    var b2 = b & 0xFF;
+
+    var invt = 1.0 - t;
+    return 0xFF000000 | 
```

Comment:
Performance could probably could be improved on by using fixed point math here, depending on what the JIT compiler does.